.pre-commit-config.yaml

```diff
@@ -17,12 +17,12 @@ repos:
         types: [python]
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":11,"endLine":17,"prLink":"https://app.graphite.dev/github/pr/yaq-project/yaqd-gdrive/1"}} -->
<i class='graphite__hidden'>[Re: lines 11 to 17]<br/></i>

```suggestion
    hooks:
      - id: black
```<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/yaq-project/yaqd-gdrive/1?utm_source=unchanged-line-comment">Graphite</a>.</span>